src/openforms/js/components/admin/index.js

```diff
@@ -13,7 +13,7 @@ import {FormCreationForm} from './form_design/form-creation-form';
 import FormVersionsTable from './form_versions/FormVersionsTable';
 import {getIntlProviderProps} from './i18n';
 import './plugin_configuration';
-import setSeleniumMarker from './selenium';
+import setE2EMarker from './selenium';
```

Comment:
Maybe this file should also be renamed to e2e instead of selenium?

Reply:
done!